tests/integration/suite/test_istio.py

```diff
@@ -66,6 +66,45 @@ def test_destination_rule(admin_pc):
     client.delete(ns)
 
 
+@pytest.mark.nonparallel
+def test_destination_rule_on_cookie(admin_pc):
```

Comment:
a small summary here to explain what this addresses or why it's being tested would be good